pynestml/codegeneration/builder.py

```diff
@@ -44,8 +57,29 @@ def __init__(self, target, options: Optional[Mapping[str, Any]]=None):
             self._target = ""
             raise InvalidTargetException()
 
+        self._redirect_output = copy.deepcopy(self.__class__._redirect_build_output)
         self._target = target
 
-    @abstractmethod
     def build(self) -> None:
+        if (self._redirect_output["default"]):
+            self.main()
+        else:
```

Comment:
Can we change the if-then-else logic here so ``build_()`` is unconditionally called and we can get rid of the "else" entirely?